### PR TITLE
Add maintainers doc

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+is maintained by the maintainers of the respective project. For example, refer
+to the maintainers specified in Parca Agent's
+[MAINTAINERS.md](https://github.com/parca/parca-agent/blob/main/MAINTAINERS.md)
+file for the Parca Agent.
+
+Note that the documentation for the Parca server is located in the [parca.dev website](https://www.parca.dev/docs/overview).
+
+Refer to the following maintainers with their focus areas:
+
+* Frederic Branczyk <https://github.com/brancz> @brancz: Everything related to the Parca (storage, backend, frontend).
```
Comment (brancz) [inline MAINTAINERS.md:11]: Everything related to the Parca server?

or

Everything related to the Parca project?

File: docs/governance.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+In case a member leaves, the [offboarding](#offboarding) procedure is applied.
+
+The current team members are:
+
+* Frederic Branczyk ([Polar Signals](https://polarsignals.com))
+* Thor Hansen ([Polar Signals](https://polarsignals.com))
+* Matthias Loibl ([Polar Signals](https://polarsignals.com))
+* Kemal Akkoyun ([Polar Signals](https://polarsignals.com))
+* Sumera Priyadarsini ([Polar Signals](https://polarsignals.com))
```
Comment (brancz) [inline docs/governance.md:71]: Yomi is a team member and maintainer of frontend

File: docs/governance.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+
+The new member is
+
+* added to the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR.
+* announced on the [developers mailing list][devs] by an existing team member. Ideally, the new member replies in this
+  thread, acknowledging team membership.
+* added to the [GitHub organization][gh] as _Owner_.
+* optionally added to the community, junkyard, and related organizations and repositories.
+* added to the [team mailing list][team].
+* [WHEN EXISTS] added to the parca.dev GSuite account with a user name of the new member's choice. (Most importantly,
```
Comment (brancz) [inline docs/governance.md:216]: we don't have a parca gsuite

File: docs/governance.md (hunk 2 above)
Comment (kakkoyun) [reply docs/governance.md]: :see_no_evil: Sorry that I've missed it

Comment (kakkoyun) [reply docs/governance.md]: He wasn't on the parca-team that's why I've missed it. I'm adding him to both.

**Comment by brancz** [comment]
lgtm

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+[MAINTAINERS.md](https://github.com/parca/parca-agent/blob/main/MAINTAINERS.md)
+file for the Parca Agent.
+
+Note that the documentation for the Parca server is located in the [parca.dev website](https://www.parca.dev/docs/overview).
+
+Refer to the following maintainers with their focus areas:
+
+* Frederic Branczyk <https://github.com/brancz> @brancz: Everything related to the Parca Project (storage, backend, frontend, agent).
+* Thor Hansen <https://github.com/thorfour> @thorfour: Storage and backend. (Please do not disturb him with questions about the frontend).
+* Matthias Loibl <https://github.com/metalmatze> @metalmatze: Everything related to the Parca (storage, backend, frontend).
```
Comment (brancz) [inline MAINTAINERS.md:13]: _the Parca server_

**Comment by kakkoyun** [comment]
@brancz Addressed all the comments. Also see https://github.com/parca-dev/docs/pull/121

**Pull request merged**